src/app/shared/form/builder/ds-dynamic-form-ui/models/array-group/dynamic-form-array.component.html

```diff
@@ -13,15 +14,17 @@
              cdkDrag
              cdkDragHandle
              [cdkDragDisabled]="dragDisabled"
-             [cdkDragPreviewClass]="'ds-submission-reorder-dragging'">
+             [cdkDragPreviewClass]="'ds-submission-reorder-dragging'"
+             [class.grey-background]="model.isInlineGroupArray">
           <!-- Item content -->
-          <div class="drag-handle" [class.drag-disable]="dragDisabled" tabindex="0">
-            <i class="drag-icon fas fa-grip-vertical fa-fw" [class.drag-disable]="dragDisabled" ></i>
+          <div class="drag-handle" [class.invisible]="dragDisabled" tabindex="0">
```

Comment:
please revert this change otherwise you're reverting a recent fix made here https://github.com/DSpace/dspace-angular/commit/9019b809939868a48d59936ad35b96ab15fcc685